vote/cnab/app/Makefile

```diff
@@ -0,0 +1,31 @@
+CNAB_ACTION ?= status
+CNAB_INSTALLATION_NAME ?= vote
+
+CNAB_P_HELM_OPTIONS ?= --dry-run --debug
+
+install:
+	@echo "Do $(CNAB_ACTION) for $(CNAB_INSTALLATION_NAME)"
+	@for chart in $$(ls charts); do [[ ! -d charts/$$chart ]] || \
+		helm install $(CNAB_P_HELM_OPTIONS) -n $(CNAB_INSTALLATION_NAME)-$$chart charts/$$chart ;\
+	done
+	@echo All charts installed
+
+uninstall: 
+	@echo "Do Uninstall"
+	@for chart in $$(ls charts); do [[ ! -d charts/$$chart ]] || \
+		helm delete $(CNAB_P_HELM_OPTIONS) $(CNAB_INSTALLATION_NAME)-$$chart ;\
```

Comment:
Thoughts on adding a `--purge` to this delete command so that the release name can be re-used?

Reply:
Good idea!